core/proto.go

```diff
@@ -44,10 +45,15 @@ func DutyFromProto(duty *pbv1.Duty) Duty {
 }
 
 // ParSignedDataFromProto returns the data from a protobuf.
-func ParSignedDataFromProto(typ DutyType, data *pbv1.ParSignedData) (ParSignedData, error) {
-	// TODO(corver): This can panic due to json unmarshalling unexpected data.
-	//  For now, it is a good way to catch compatibility issues. But we should
-	//  recover panics and return an error before launching mainnet.
+func ParSignedDataFromProto(typ DutyType, data *pbv1.ParSignedData) (odata ParSignedData, oerr error) {
```

Comment:
where is `odata` returned?

Reply:
nowhere, but I think I cannot omit one while having names on the others..